src/routers/user.router.ts

```diff
@@ -35,23 +35,7 @@ const userController: Router = express.Router();
 userController.use(AuthMiddleware);
 noAuthUserController.use('/session', userController);
 
-
-async function initSession(user: UserEntity): Promise<{ id: number; did: string, appToken: string, username?: string, displayName: string, privateData: string }> {
-	const secret = new TextEncoder().encode(config.appSecret);
-	const appToken = await new SignJWT({ did: user.did })
-		.setProtectedHeader({ alg: "HS256" })
-		.sign(secret);
-	return {
-		id: user.id,
-		appToken,
-		did: user.did,
-		displayName: user.displayName || user.username,
-		privateData: user.privateData.toString(),
-		username: user.username,
-	};
-}
-
-async function filterUserData(user: UserEntity): Promise<{ id: number, did: string, appToken: string, username?: string, displayName: string, privateData: string, webauthnUserHandle: string, webauthnCredentials: WebauthnCredentialEntity[] }> {
+async function initSession(user: UserEntity): Promise<{ id: number, did: string, appToken: string, username?: string, displayName: string, privateData: string, webauthnUserHandle: string, webauthnCredentials: WebauthnCredentialEntity[] }> {
 	const secret = new TextEncoder().encode(config.appSecret);
 	const appToken = await new SignJWT({ did: user.did })
 		.setProtectedHeader({ alg: "HS256" })
```

Comment:
It looks like `webauthnCredentials` is no longer needed in the frontend since we removed client-side WebAuthn verification (https://github.com/wwWallet/wallet-frontend/pull/264/commits/184afa200f053c82a342c23c598d66f8de7ae167). Should we drop it?